src/resource/index.test.js

```diff
@@ -78,6 +78,17 @@ describe('Resource', () => {
       assert.equal(badEncodingRe.test($.html()), false);
       assert.equal(typeof $, 'function');
     });
+
+    it('doesnt mangle non-ascii characters in prefetched response', async () => {
+      const url = 'https://www.gruene.de/themen/digitalisierung';
+      const prefetched =
+        '<!DOCTYPE html><html lang="de"><head><meta charSet="UTF-8"/><meta http-equiv="x-ua-compatible" content="ie=edge"/><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/></head><body><div id="___gatsby"><h1 class="styles-module--headline--niWjO">Wir gestalten die Digitalisierung</h1><p>Wir Grüne kämpfen für ein offenes, gemeinwohlorientiertes Netz. Wir wollen den digitalen Wandel gerecht gestalten und setzen uns für Verantwortung, Freiheit und Recht im Netz ein. Netzpolitik und Digitalisierung sind zentrale politische Querschnittsaufgaben für eine moderne Gesellschaft. Im Mittelpunkt stehen für uns eine zukunftsfähige digitale Infrastruktur, der freie und gleichberechtigte Zugang zum Netz für alle, der Schutz unserer Privatsphäre und persönlichen Daten, sowie eine modernisierte  Verwaltung.</p></div></body></html>';
+
+      const $ = await Resource.create(url, prefetched);
+      console.log($.html());
```

Comment:
Really liking this solution to the double encoding issue.  Let's remove this console.log from the test and I believe we are all set.

Reply:
Whoops!!  I just removed it.